source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -105,6 +102,10 @@ def get_street_network_routing_matrix(
             origins, destinations, mode, max_duration, speed_switcher, **kwargs
         )
 
+        # asgard doesn't know what to do with 'car_no_park'...
+        if mode == FallbackModes.car_no_park.name:
```

Comment:
why we don't upgrade **Asgard** to know **car_no_park** ?

Reply:
From the point of view of asgard, car and car_no_park is the same thing...